content/en/docs/reference/commands/jx_create_cluster.md

```diff
@@ -30,8 +30,7 @@ Depending on which cloud provider your cluster is created on possible dependenci
 
   * kubectl (CLI to interact with Kubernetes clusters)  
   * helm (package manager for Kubernetes)  
-  * draft (CLI that makes it easy to build applications that run on Kubernetes)  
-  * virtualisation drivers (to run Minikube in a VM)  
+  * draft (CLI that makes it easy to build applications that run on Kubernetes)
```

Comment:
this change you will need to make in the `jx` codebase. The command reference is generated from the `jx` source. see https://github.com/jenkins-x/jx/blob/cff0301241c853ce8d68bddb3009f6585d39ef97/pkg/cmd/create/create_cluster.go#L68

For what it's worth, it is the version stream where the pull request for the doc changes is generated - https://github.com/jenkins-x/jenkins-x-versions/blob/master/jx/scripts/update-jx-website.sh#L29

Reply:
Good catch :+1:, will remove